server/cmwell-cons/src/main/scala/ctl.scala

```diff
@@ -1616,7 +1614,11 @@ abstract class Host(user: String,
     // create kafka topics
     val replicationFactor = math.min(hosts.size, 3)
     // scalastyle:off
-    val createTopicCommandPrefix = s"cd ${instDirs.globalLocation}/cm-well/app/kafka/cur; export PATH=/opt/cm-well/app/java/bin:$$PATH ; sh bin/kafka-topics.sh --create --zookeeper ${pingAddress}:2181 --replication-factor $replicationFactor --partitions ${hosts.size} --topic"
+    val createTopicCommandPrefix =
+      if (Files.exists(Paths.get(s"${instDirs.globalLocation}/cm-well/app/java")))
```

Comment:
Instead of duplicating the whole command, just use string interpolation and conditionally add the `JAVA_HOME` section.